Example/ViewController.m

```diff
@@ -277,52 +276,24 @@ - (void)getSessionData
     }];
 }
 
-- (DMEScope *)createSampleScopeForOneYearOfSocialData
+- (DMEPullConfiguration *)createSampleConfiguration
```

Comment:
Given we moved the configuration to this method now - let's move this method to the top, so that it is more easily visible to new developers :)